exercises/practice/knapsack/tests/Tests.elm

```diff
@@ -0,0 +1,85 @@
+module Tests exposing (tests)
+
+import Expect
+import Knapsack exposing (..)
+import Test exposing (..)
+
+
+tests : Test
+tests =
+    describe "Knapsack"
+        [ test "no items" <|
+            \() ->
+                Expect.equal 0 <| maximumValue 100 []
```

Comment:
We usually format our tests in this way
```suggestion
            maximumValue 100 []
            |> Expect.equal 0
```
It feels more like English.
(this applies to all tests)